README.md

```diff
@@ -81,6 +81,8 @@ It starts etcd and simulator-server locally.
 
 To run the frontend, please see [README.md](./web/README.md) on ./web dir.
 
+Read more about differnet environment variables being used in simulator server [here.](./docs/env-variables.md)
```

Comment:
```suggestion
Read more about environment variables being used in simulator server [here.](./docs/env-variables.md)
```

Reply:
And please move it to L48 since these configurations is not only related to simulator frontend.

Reply:
Updated.